.idea/dash-draggable.iml

```diff
@@ -2,7 +2,7 @@
 <module type="PYTHON_MODULE" version="4">
   <component name="NewModuleRootManager">
     <content url="file://$MODULE_DIR$" />
-    <orderEntry type="inheritedJdk" />
+    <orderEntry type="jdk" jdkName="Python 3.6 (dash-draggable)" jdkType="Python SDK" />
```

Comment:
We should probably remove the entire `.idea/` folder. And add it to gitignore so it's not accidentally added.

Reply:
agreed, sorry, I forget about it